testing-unit/src/main/java/com/bumble/appyx/testing/unit/helper/ParentNodeTestHelper.kt

```diff
@@ -0,0 +1,34 @@
+package com.bumble.appyx.testing.unit.helper
+
+import androidx.lifecycle.Lifecycle
+import com.bumble.appyx.core.children.nodeOrNull
+import com.bumble.appyx.core.node.ParentNode
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+
+fun <Routing : Any, N : ParentNode<Routing>> N.parentNodeTestHelper() =
+    ParentNodeTestHelper(this)
+
+class ParentNodeTestHelper<Routing : Any, N : ParentNode<Routing>>(
+    private val node: N
+) : NodeTestHelper<N>(
+    node = node
+) {
+
+    fun <Routing : Any> assertChildHasLifecycle(routing: Routing, state: Lifecycle.State) {
+        node.children.value.keys
```

Comment:
If we fail to find the child, no assertion is run. Perhaps you could tweak this function so that it throws an exception in some manner?

Otherwise tests might start passing when they shouldn't.

Reply:
good point, fixed.